registration/registration.go

```diff
@@ -88,6 +88,7 @@ func (r *OpenConsumerChannelRequest) GetMessage() []byte {
 	message := []byte{}
 	message = append(message, crypto.Pad(math.U256Bytes(big.NewInt(r.ChainID)), 32)...)
 	message = append(message, common.HexToAddress(r.HermesID).Bytes()...)
+	message = append(message, common.HexToAddress(r.RegistryAddress).Bytes()...)
```

Comment:
check [the Sc call](https://github.com/mysteriumnetwork/payments-smart-contracts/blob/de2b51cabba3224dc2d6837b61624b996ed522cc/contracts/Registry.sol#L111) I think the order is chainID, registry, hermes, transactorFee

Reply:
Thanks. I didn't know order is important